### PR TITLE
(fix) fix logic for hiding/showing action buttons set

### PR DESCRIPTION
**Pull request by arodidev**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary

This fixes a bug introduced by [this PR](https://github.com/openmrs/openmrs-form-engine-lib/pull/227) where the button set was hiding when the form had less than one scrollable page.

## Screenshots

https://github.com/openmrs/openmrs-form-engine-lib/assets/51090527/ba6551a5-8d91-48c3-8f4d-87eb5df633c5

https://github.com/openmrs/openmrs-form-engine-lib/assets/51090527/95d40e25-faad-459e-bbb6-3ad50f196190

https://github.com/openmrs/openmrs-form-engine-lib/assets/51090527/53cd8bf3-599b-4a0b-be76-d81da65b9b1c

### CODE REVIEW

File: src/form-engine.component.tsx [hunk 1]
```diff
@@ -118,7 +118,8 @@ const FormEngine: React.FC<FormProps> = ({
   }, [patient?.id, sessionMode, workspaceLayout]);
 
   const showButtonSet = useMemo(() => {
-    return sessionMode !== 'embedded-view' && workspaceLayout === 'minimized' && scrollablePages.size <= 1;
+    if (sessionMode == 'embedded-view') return false;
+    return workspaceLayout === 'minimized' || ('maximized' && scrollablePages.size <= 1);
```
Comment (denniskigen) [inline src/form-engine.component.tsx:122]:
```suggestion
    if (sessionMode === 'embedded-view') {
      return false;
    }
    
    return workspaceLayout === 'minimized' || (workspaceLayout === 'maximized' && scrollablePages.size <= 1);
```

Comment (denniskigen) [reply src/form-engine.component.tsx]: Do the updated changes work in Form Render Test as well? I've tested that they work in the Patient Chart:

CleanShot 2024-05-02 at 12  20 23@2x

**Review by denniskigen — APPROVED** [review]
Thanks, Jamie!

File: src/form-engine.component.tsx (hunk 1 above)
Comment (arodidev) [reply src/form-engine.component.tsx]: Thanks @denniskigen, works just fine.

**Pull request merged**